_includes/configure-registry-json.md

```diff
@@ -2,63 +2,46 @@
 
 ## Create a registry.json file
```

Comment:
Compared to the previous version, we've lost the information about where the file lives and what its contents are.

I'm in two minds about this. On the one hand, some users will want to know, and we've hidden it from them. On the other hand, we don't want to teach people the old, manual way to install it.

Reply:
Maybe this is the solution. At the end of the Windows section and the Mac section, add another sentence, something like:

This will create the `registry.json` file at _location_ with information about which organization the user must belong to. Make sure that this file is not editable by the individual developer, only by the system administrator.

(I don't think we need to specify the contents of the file — they can look inside it if they need to know — but the location and file permission requirements would still be helpful).